src/OptimalTransport.jl

```diff
@@ -239,97 +239,227 @@ function sinkhorn2(μ, ν, C, ε; regularization=false, plan=nothing, kwargs...)
 end
 
 """
-    sinkhorn_unbalanced(mu, nu, C, lambda1, lambda2, eps; tol = 1e-9, max_iter = 1000, verbose = false, proxdiv_F1 = nothing, proxdiv_F2 = nothing)
+    sinkhorn_unbalanced(
+        μ, ν, C, λ1::Real, λ2::Real, ε;
+        atol=0, rtol=atol > 0 ? 0 : √eps, check_convergence=10, maxiter=1_000,
+    )
+
+Compute the optimal transport plan for the unbalanced entropic regularization optimal
```

Comment:
I think it's better to call it the `unbalanced entropic regularized optimal transport problem`.

Reply:
Ah yeah, this is just copied from POT, I can change it.

Reply:
I noticed that also in other docstrings we use the term `entropic regularization optimal transport problem` (copied from POT as well). Do you think it should be changed there as well (in a separate PR) or should we keep `unbalanced entropic regularization optimal transport problem`? I always thought the term is a bit odd even though it's used in POT, so I would be fine with changing it in the other places as well.

Reply:
I'd also go with changing in all the occurrences. I think it might be a typo on `POT.py` or something, because in the literature I don't remember anyone calling it like that. Usually they either call `etropic optimal transport` or `entropic regularized optimal transport`.

Reply:
I went with `entropically regularized OT` now, I've seen this as well and I think I prefer it over `entropic regularized OT`.

Reply:
Ok. I'm fine with both :)